cmd/capture.go

```diff
@@ -49,7 +49,7 @@ by passing the function name symbol and the binary args.
 		for _, functionSymbol := range functionSymbolList {
 			syscalls, err := captor.Capture(functionSymbol, args, captureOpts)
 			if err != nil {
-				return fmt.Errorf("error capturing syscall: %w", err)
+				fmt.Printf("error capturing syscall: %w", err)
```

Comment:
You could consider using

Either errors.Join to gather them all

Or fmt.Fprintf to send the errors to stderr


```suggestion
				fmt.Fprintf(os.Stderr, "error capturing syscall: %v", err)
```